metadata/pdf_get_custom_xmp_metadata.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/core"
+	"github.com/unidoc/unipdf/v3/model"
+	"github.com/unidoc/unipdf/v3/model/xmputil"
+
+	xmprights "github.com/trimmer-io/go-xmp/models/xmp_rights"
+)
+
+const licenseKey = `
+-----BEGIN UNIDOC LICENSE KEY-----
+Free trial license keys are available at: https://unidoc.io/
```

Comment:
Please update to metered key, all our examples use that.  (Can copy paste from other examples)

Reply:
Ah, I've actually copied from the only one that didn't use metered license.
Fixed all examples that had a normal license.